source/Mlos.Python/mlos/Optimizers/long_haul_tests/LongHaulTestBayesianOptimizer.py

```diff
@@ -115,8 +115,7 @@ def test_bayesian_optimizer_on_simple_2d_quadratic_function_pre_heated(self):
 
             # Register the observation with the optimizer
             bayesian_optimizer.register(input_values_df, target_values_df)
-
-        self.logger.info(f"Optimum: {bayesian_optimizer.optimum()}")
+        self.logger.info(f"Optimum: {bayesian_optimizer.optimum()[1]}")

```

Comment:
Could we please change it to:
```
best_config_point, best_objective = bayesian_optimizer.optimum()
self.logger.info(f"Optimum: {best_objective}")
```
Unpacking the tuple this way makes it clear to the reader what's going on. It also creates a neat pattern for people to copy-paste in their own client code.

Reply:
or even:

```
best_config_point, best_objective = bayesian_optimizer.optimum()
self.logger.info(f"Optimum config: {best_config_point}, optimum objevtive: {best_objective}")
```

Reply:
the linter will complain about the unused variable, though, right?
I had it first as 
```python
_, best_objective = bayesian_optimizer.optimum()
```
but I'm not sure that's more useful?